2019-12-06 15:56:18,226 ERROR: Exception on /reset_password_request [POST] [in /Users/scammons/Projects/learn_flask/venv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/scammons/Projects/learn_flask/venv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/scammons/Projects/learn_flask/venv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/scammons/Projects/learn_flask/venv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/scammons/Projects/learn_flask/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/scammons/Projects/learn_flask/venv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/scammons/Projects/learn_flask/venv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/scammons/Projects/learn_flask/app/routes.py", line 128, in reset_password_request
    send_password_request_email(user)
  File "/Users/scammons/Projects/learn_flask/app/email.py", line 19, in send_password_request_email
    user=user, token=token))
TypeError: send_email() missing 1 required positional argument: 'sender'
2019-12-06 15:56:49,996 INFO: Microblog startup [in /Users/scammons/Projects/learn_flask/app/__init__.py:49]
